prime-router/src/test/kotlin/fhirengine/engine/FhirTranslatorTests.kt

```diff
@@ -48,7 +48,7 @@ import kotlin.test.Test
 
 private const val ORGANIZATION_NAME = "co-phd"
 private const val RECEIVER_NAME = "full-elr-hl7"
-private const val ORU_R01_SCHEMA = "metadata/hl7_mapping/ORU_R01/ORU_R01-base"
+private const val ORU_R01_SCHEMA = "metadata/hl7_mapping/receivers/STLTs/CA/CA"
```

Comment:
This change is to get the test that verifies we can handle generating 2.5.1 messages when the message encoding contains 5 characters.  The default ORU_R01 schema now encodes to 27